dex/networks/dgb/params.go

```diff
@@ -54,6 +55,7 @@ var (
 	})
 	// TestNetParams are the clone parameters for testnet.
 	TestNetParams = btc.ReadCloneParams(&btc.CloneParams{
+		Name:             "testnet4",
```

Comment:
None of these `Name`s are necessary, they are just for completion, and debugging.  Initially I wanted to have the `"chain"` string from `getblockchaininfo` compared against these values that we define as expected, but that's not where the nodes get the string to return in that field, so we just have the dumb switch with "main"/"test"/reg" instead.